images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -100,6 +100,7 @@ $toolsList = @(
     (Get-CodeQLBundleVersion),
     (Get-DockerVersion),
     (Get-DockerComposeVersion),
+    (Get-GDKVersion),
```

Comment:
You should add a condition(Test-IsWin19) to get version only for Windows Server 2019.

Reply:
Good catch. Updated